### DISCUSSION TITLE
can't find memhook link

### DISCUSSION THREAD

**Discussion by XCloudAI** [openucx/ucx#8325] [Q&A] [answered]
In below UCX/wiki page
 https://github.com/openucx/ucx/wiki/Malloc-hooks
it gives the link page of valgrind malloc hook code.
 http://code.google.com/p/valgrind-variant/source/browse/trunk/valgrind/coregrind/m_replacemalloc/vg_replace_malloc.c#1175
However, the link seems not usable.
Could the community update the right link? Or tell what right is needed to see the code?

**Comment by yosefe** [comment, marked as answer]
fixed
